pandas/_libs/tslibs/util.pxd

```diff
@@ -227,3 +236,37 @@ cdef inline bint is_nan(object val):
     is_nan : bool
     """
     return (is_float_object(val) or is_complex_object(val)) and val != val
+
+
+cdef inline const char* get_c_string_buf_and_size(object s,
+                                                  Py_ssize_t *length):
+    """
+    Extract internal char * buffer of unicode or bytes object `s` with
```

Comment:
is there an extra space either before or after the `*`?

Reply:
Fix this